cobigen-cli/src/main/java/com/devonfw/cobigen/cli/utils/CobiGenUtils.java

```diff
@@ -234,7 +236,12 @@ public void registerPlugins() {
 
             File cpFile = rootCLIPath.resolve(MavenConstants.CLASSPATH_OUTPUT_FILE).toFile();
             if (!cpFile.exists()) {
-                buildCobiGenDependencies(pomFile);
+                try {
+					buildCobiGenDependencies(pomFile);
+				} catch (IOException e) {
+					// TODO Auto-generated catch block
+					e.printStackTrace();
```

Comment:
Please add a proper `LOG.error("....",e)` message that explains why this error has occurred (the context, whether the user can do something to fix it...)